src/internal/_stepCat.js

```diff
@@ -20,7 +22,7 @@ module.exports = (function() {
   var _stepCatObject = {
     '@@transducer/init': Object,
     '@@transducer/step': function(result, input) {
-      return merge(
+      return _objectAssign(
```

Comment:
Prehistory as in Eweda? I'm sure there was also some shallow cloning utility at some point...

Funny enough, I was about to change ``merge`` to use ``_objectAssign`` before I posted this patch. Then I saw there are few levels of abstractions there and it's not worth it. On the other hand, it's probably a good idea to that in ``mergeAll``.